components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -67,9 +64,6 @@ service ReportingService {
 			resource: "compliance:reporting:report-ids"
 			action: "compliance:reportids:list"
 		};
-		option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
-			tags: "Reports";
-		};
```

Comment:
As a result of this unnecessary level of redirection, we forgot to update the tag of this endpoint (`ListReportIds`) so it wasn't being displayed when viewing the API docs in the browser.